src/java.base/share/classes/java/io/Console.java

```diff
@@ -390,13 +391,31 @@ public void flush() {
         pw.flush();
     }
 
+
+    /**
+     * Returns the {@link java.nio.charset.Charset Charset} object used in
+     * the {@code Console}.
```

Comment:
What would you think about re-phrasing the first sentence to use "for the Console" rather than "in the Console".

Reply:
Changed to "for the Console", as well as `@return`.